neqo-server/src/main.rs

```diff
@@ -382,6 +386,7 @@ impl ServersRunner {
                     &[args.alpn.clone()],
                     anti_replay,
                     cid_mgr,
+                    args.max_streams_bidi,
```

Comment:
It would be better if this were passed ConnectionParameters.  You can add a function to Args to generate a set.  Then it will be easier to add new options.